pycds/manage_views.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from pycds.materialized_view_helpers import MaterializedViewMixin
+from pycds.weather_anomaly import \
+    DailyMaxTemperature, DailyMinTemperature, \
+    MonthlyAverageOfDailyMaxTemperature, MonthlyAverageOfDailyMinTemperature, \
+    MonthlyTotalPrecipitation
+
+daily_views = [DailyMaxTemperature, DailyMinTemperature]
+monthly_views = [MonthlyAverageOfDailyMaxTemperature, MonthlyAverageOfDailyMinTemperature, MonthlyTotalPrecipitation]
+
+logger = logging.getLogger(__name__)
+
+
+def manage_views(session, operation, which_views):
```

Comment:
The `which_views` parameter is a little confusing. What should the parameter value be? A string? An index? From looking at the usage elsewhere, I see that it should be one of the keys below ('daily'|'monthly-only'|'all'), but that's not necessarily clear. I would maybe change the parameter to be named `which_set` *singular* because you are only selecting one thing. A function docstring would be nice too :)